### PR TITLE
Semaphore Community Grant Round

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

New form for the Semaphore Community Grant Round.

## Related Issue

#233

### CODE REVIEW

File: src/pages/api/semaphore-grant.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        res.status(400).json({ status: 'fail' });
+        return;
+      }
+
+      // send submission data to a google spreadsheet
+      try {
+        await addRowToSpreadsheet(
+          {
+            id: '1YpA7U94D9t7wV4FabRbKJLB8GHhGpszqukhozhqLplE',
+            sheetName: 'Applications'
```
Comment (pettinarip) [inline src/pages/api/semaphore-grant.ts:107]: I had to hardcode these values since Netlify is giving me the following error:
```
Your environment variables exceed the 4KB limit imposed by AWS Lambda. Please consider reducing them.
```
Those vars are not sensitive information since you need the proper credentials to access that doc.

Perhaps we could do the same for the other lambda functions to avoid having this error in the future.

cc @samajammin

Comment (samajammin) [reply src/pages/api/semaphore-grant.ts]: Ya I support reviewing our ENV variables to see which must be private. Spreadsheet ID & name don't seem necessary.

**Review by nhsz — COMMENTED** [review]
@pettinarip Looking awesome, great job! Just left a question

File: src/components/Nav.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ export const Nav: FC = () => {
           <Image src={logoSVG} alt='Ecosystem Support Program logo' height={64} width={148} />
         </Box>
 
-        <Box display={{ base: 'block', md: 'none' }}>
+        <Box display={{ base: 'block', md: 'none' }} zIndex={9999}>
```
[the remark this answers is not in the document]
Comment (pettinarip) [reply src/components/Nav.tsx:49]: Good question. Those fixes on the `zIndex` were added to support the workaround I had to do to display a background on the Header position.
Due to how the header & the bg image is positioned, I couldn't set a background color to the header element itself. If you take a look at `SemaphoreGrantLayout`, you will see that I'm not actually adding the gradient to the header. I'm adding it to an empty `div` called `DimLayer` that is simulating the size of the header.
Basically, to be able to display that `DimLayer` in between the backgound image and the logo & menu, I had to adjust those z-indexes.

File: src/pages/semaphore-grants/index.tsx [hunk 3]
```diff
@@ -0,0 +1,485 @@
...
+                          fontWeight={700}
+                          color='brand.orange.100'
+                          href='https://interep.link/'
+                          isExternal
+                          _hover={{ textDecoration: 'none' }}
+                        >
+                          Interep
+                        </Link>{' '}
+                        uses zero knowledge proofs to verify reputation from an existing account
+                        such as Github or Twitter without retaining any identifying information.
```
Comment (corwintines) [inline src/pages/semaphore-grants/index.tsx:224]:
```suggestion
                        such as GitHub or Twitter without retaining any identifying information.
```

File: src/pages/semaphore-grants/index.tsx [hunk 4]
```diff
@@ -0,0 +1,485 @@
...
+                </section>
+              </Stack>
+
+              <Stack>
+                <section id='semaphore-as-privacy-layer' ref={ref3}>
+                  <PageSection mb={6}>Semaphore as a privacy layer</PageSection>
+
+                  <PageText mb={6}>
+                    Semaphore is designed to be a simple and generic privacy layer for decentralized
+                    applications (dApps) on Ethereum.{' '}
```
Comment (corwintines) [inline src/pages/semaphore-grants/index.tsx:123]:
```suggestion
                    applications (dapps) on Ethereum.{' '}
```

Based on the style guide we use on ethereum.org I think we should change this, but dont think this is a blocker.

**Review by corwintines — DISMISSED** [review]
Looks good to me

**Review by corwintines — APPROVED** [review]
